openshift/ci-operator/build-image/elasticsearch-subscription.yaml

```diff
@@ -7,5 +7,4 @@ spec:
   source: elasticsearch
   sourceNamespace: openshift-operators
   name: elasticsearch-operator
-  startingCSV: elasticsearch-operator.v0.0.1
   channel: preview
```

Comment:
Is it ok having v1alpha1 intact?
`elasticsearch-subscription.yaml:apiVersion: operators.coreos.com/v1alpha1`

The same question for cluster-logging-subscription.yaml...
`cluster-logging-subscription.yaml:apiVersion: operators.coreos.com/v1alpha1`

Reply:
> Is it ok having v1alpha1 intact?
> `elasticsearch-subscription.yaml:apiVersion: operators.coreos.com/v1alpha1`
> 
> The same question for cluster-logging-subscription.yaml...
> `cluster-logging-subscription.yaml:apiVersion: operators.coreos.com/v1alpha1`

Yes.  For some reason, the subscription api is still v1alpha1

Reply:
i would assume yes, i would have expected that in conjunction w/ the marketplace api changes to v1.  

@ecordell @jwforres can you weigh in?

Reply:
If so, we'll have to open another PR, if/when that happens . . .